### PR TITLE
SALTO-4011/add display name to error

### PR DESCRIPTION
**Pull request by guyhamo**
add display names of missing users to the error message.

---

_Additional context for reviewer_
tested with cloud and dc
cloud has both values in this format id: "display name", ....
dc is not changed with just id.

---
_Release Notes_: 
jira adapter:
* add display name to user errors with visible id.

---
_User Notifications_:

### CODE REVIEW

File: packages/jira-adapter/src/change_validators/account_id.ts [hunk 1]
```diff
@@ -154,7 +162,7 @@ const checkAndAddChangeErrors = (
   }
 
   if (!Object.prototype.hasOwnProperty.call(userMap, accountId)) {
-    missingUsers.add(accountId)
+    missingUsers.add({ accountId, displayName: currentDisplayName })
```
Comment (alonstern) [inline packages/jira-adapter/src/change_validators/account_id.ts:165]: Objects don't work inside sets, meaning it won't remove duplications as you add them, you will need to add them to a list and then use something like `_.uniqBy`

Comment (guyhamo) [reply packages/jira-adapter/src/change_validators/account_id.ts]: 👍

File: packages/jira-adapter/src/change_validators/account_id.ts [hunk 2]
```diff
@@ -196,24 +207,25 @@ const createChangeErrorsForAccountIdIssues = (
       missingDisplayNamePaths
     ), config) })
 
+  const missingUsersSet = new Set(_.uniqBy(missingUsers, formatMissingUser))
```
Comment (alonstern) [inline packages/jira-adapter/src/change_validators/account_id.ts:210]: no reason to work with a set now, I think you can just pass it as a list in the filter

**Review by alonstern — APPROVED** [review]
👌

File: packages/jira-adapter/src/change_validators/account_id.ts (hunk 2 above)
Comment (guyhamo) [reply packages/jira-adapter/src/change_validators/account_id.ts:210]: 👍

**Pull request merged**